### DISCUSSION TITLE
캐릭터 생성 위치를 수정할래요! [치자피즈]

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#196] [3. Creator Samples]
우선 제페토 멀티플레이 프로젝트를 기준으로 시작합니다. 기본 설정이 좋아요! (이걸 기준으로 공부를 시작해요!)
https://github.com/naverz/zepeto-multiplay-example

또는 프로젝트 초기 셋팅을 위한 짧은강의를 만들어보았어요 :l (오래전에 만들어둔 자료라 어색하지만 누군가에게 조금이라도 도움이 될 수 있을까요?)
https://cheesafizz.notion.site/1-b04332d8a4024f24a49a10b593b03c1c

클라이언트 스타터에서 캐릭터 생성부분에서 스폰인포를 설정하는걸로 간단하게 원하는 스폰 포인트를 만들 수 있어요!

<img width="1026" alt="스크린샷 2022-01-10 오후 8 37 08" src="https://user-images.githubusercontent.com/90679926/148760723-cde8d93b-a427-4070-a958-f296bcd5d9f0.png">

슬슬 저희 개발자 톡방의 질문이 고갈되어가요, 필요한 기능이나 예제가 있을까요?

틀린 정보가 있다면 댓글달아주세요 :D

**Comment by hipsc159** [comment]
정말 구세주시네요.. 감사합니다!
